DuckDuckGo/Tab/View/BrowserTabViewController.swift

```diff
@@ -466,16 +466,17 @@ final class BrowserTabViewController: NSViewController {
             onFireButtonPressed: { [weak delegate] in
                 delegate?.dismissViewHighlight()
             })
+        containerStackView.spacing = 0
         let hostingController = NSHostingController(rootView: AnyView(daxView))
-
         daxContextualOnboardingController = hostingController
         insertChild(daxContextualOnboardingController!, in: containerStackView, at: 0)
         hostingController.view.translatesAutoresizingMaskIntoConstraints = false
         NSLayoutConstraint.activate([
+            hostingController.view.topAnchor.constraint(equalTo: containerStackView.topAnchor),
```

Comment:
Seems that this is the same constraints at line 477

Reply:
questi non ci dovrebbero essere